log-replication/src/main/java/org/corfudb/logreplication/runtime/CorfuLogReplicationRuntime.java

```diff
@@ -77,6 +78,23 @@ public CorfuLogReplicationRuntime(@NonNull LogReplicationRuntimeParameters param
         corfuRuntime.connect();
     }
 
+    /**
+     * Constructor
+     *
+     * @param parameters runtime parameters
+     */
+    @VisibleForTesting
```

Comment:
Nan can you please rebase again on top of log-replication-feature-annym, this code changed a bit.